### ISSUE TITLE
Issues compiling in Visual Studio 2015 with CEFSharp as a dependency

### ISSUE THREAD

**Issue by kevfromireland** [cefsharp/CefSharp#1101]
This isn't about compiling CEFSharp within VS2015, but rather consuming the CEFSharp library in a .NET 4.5 App in VS2015

I have this snippet of code in my app:

```
var browserSettings = new BrowserSettings();
browserSettings.WebSecurityDisabled = true;
```

and it compiles fine in VS2012 but not in VS2015. The error message is:

```
Severity	Code	Description	Project	File	Line
Error	CS1545	Property, indexer, or event 'BrowserSettings.WebSecurityDisabled' is not supported by the language; try directly calling accessor methods 'BrowserSettings.get_WebSecurityDisabled()' or 'BrowserSettings.set_WebSecurityDisabled(?)
````

I think this is because the type of the property is ` Nullable<bool>^` and this ref type isn't expressible in c#. Should this be a ref type?

**Comment by amaitland** [comment]
> Should this be a ref type?

As `Nullable` is a struct, it should probably be a `value` type. Though I'm surprised it causes a problem now, there hasn't been any problems with `VS2012/3`.

**Comment by amaitland** [comment]
As a workaround you should be able to use the `--disable-web-security` command line argument.

For a similar example see https://github.com/cefsharp/CefSharp/blob/master/CefSharp.Example/CefExample.cs#L37

**Comment by amaitland** [comment]
See #1102

**Comment by amaitland** [comment]
The metadata for `BrowserSettings` was previously showing `ValueType`, now it shows `bool?`.

**Comment by kevfromireland** [comment]
Yeah, this is a breaking change in VS2015. Thanks for the workaround, I'll use that for now

**Comment by amaitland** [comment]
`BrowserSettings` now uses an enum to represent the different states (`Enabled/Disabled/Default`), also renamed a number of the properties. Changes have been merged into `master`

**Comment by amaitland** [comment]
This should also have been fixed with the release of `41.0.1`

https://github.com/cefsharp/CefSharp/commit/cf3d27be842b4bee08103484b859b661cb95a6f5

**Comment by amaitland** [comment]
> BrowserSettings now uses an enum to represent the different states (Enabled/Disabled/Default), also renamed a number of the properties.

I should make note that for version `41.0.1` it's just a minor bug fix, in version `43` it's a rewrite to remove the confusion and simplify the code.
